MaxText/inference_microbenchmark.py

```diff
@@ -150,20 +150,20 @@ def ar_benchmark(config, engine, params, decode_state, global_batch_size, cache_
   )
 
   result_dict = {
-      "ar_step_in_ms": ar_average_ms,
-      "ar_step_in_ms_per_seq": ar_average_ms / global_batch_size,
-      "ar_global_batch_size": global_batch_size,
-      "ar_total_throughput_tokens_per_second": total_throughput,
-      "ar_device_bandwidth_GB_per_second": bw_per_device,
+      "step_in_ms": ar_average_ms,
+      "step_in_ms_per_seq": ar_average_ms / global_batch_size,
+      "global_batch_size": global_batch_size,
+      "total_throughput_tokens_per_second": total_throughput,
+      "device_bandwidth_GB_per_second": bw_per_device,
```

Comment:
@morgandu We probably still want the `ar_` prefix here, right? Because once we truncate the `autoregressive_` prefix from the parent level, it will still be here.

Reply:
if we don't truncate autoregressive_ from bq, we can remove the ar_, if we are truncate autoregressive_ from bq, we should keep the ar_

Reply:
Kept `autoregressive`.